src/components/Card/Card.scss

```diff
@@ -0,0 +1,14 @@
+@import "styles/colors.scss";
+
+.card {
+  color: $color-off-white;
+  font-family: "Open sans";
+
+  position: relative;
+  z-index: 2;
+
+  .inner-container {
+    background-color: $color-dark-grey;
+    margin: -0.75rem;
```

Comment:
Also should be done with tailwind.

Reply:
can't do negative margins with tailwind.

Reply:
my b, resolved